components/pdfviewer/annotations.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Annotations
+page_title: PdfViewer - Annotations
+description: Annotations in the PDF Viewer for Blazor. How to add and configure annotations in the PDF Viewer for Blazor
+slug: pdfviewer-annotations
+tags: telerik,blazor,pdf,pdfviewer,annotations
+published: True
+position: 15
+---
+
+# PdfViewer Annotations
+
+The PdfViewer supports a set of annotation types that the user can add to enhance the PDF document - highlight important text areas or add notes as free text. 
+
+The PDFViewer creates a dedicated layer for the annotations and includes them when downloading or printing the file. When downloading a file with annotations and opening it again, the PDFViewer allows editing the previously created annotations.
+
+To add and manage the annotations, the user can click on the dedicated button in the [Toolbar]({%slug pdfviewer-toolbar%}). It opens an annotations bar with the available annotation types.
+
+## Supported Annotations
+
+The PDFViewer provides the following annotation types:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Annotation Type | Description |
+| --- | --- |
+| **Text Highlight** | Allows highlighting the selected text in the document. Provides an option to select the desired highlight color. |
```

Comment:
There are two "select" words here with very different meaning. Consider removing one of them for easier understanding.

```suggestion
| **Text Highlight** | Allows highlighting the selected text in the document. Users can choose the desired highlight color. |
```